services/common/src/components/parties/OrgBookSearch.tsx

```diff
@@ -54,6 +60,14 @@ const OrgBookSearch: FC<OrgBookSearchProps> = ({ isDisabled = false, setCredenti
     setIsSearching(false);
   };
 
+  const placeHolder = "Start typing to search OrgBook...";
+
+  useEffect(() => {
+    if (selectedParty !== current_party) {
+      setSelectedParty(placeHolder);
```

Comment:
In the flow, 
1. User sees the place holder (initial step, when no party is associated)
2. Then user types and select one party and click on `Associate` button. This is will disabled the search box
3. Then (if) user want to change the selected party, user clicks on `Disassociate` button (screen shots in the PR)
4. In step (3), if assigned selected party as 
    - setSelectedParty(NONE) -> it will show None in the search box
    - setSelectedParty("") -> it will show the empty in the search box.

So I thought assigning the place holder to the party, makes things easier for the user.

If there's better options to handle this, please let me know.